warehouse/query-core/src/test/java/datawave/query/jexl/visitors/IteratorBuildingVisitorTest.java

```diff
@@ -37,6 +39,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import static datawave.query.Constants.NULL;
```

Comment:
you imported the static then typed the value everywhere

Reply:
ha, never completed that thought. Fixed.